src/sage/schemes/elliptic_curves/weierstrass_morphism.py

```diff
@@ -973,6 +973,105 @@ def inseparable_degree(self):
         """
         return Integer(1)
 
+    def is_identity(self):
+        r"""
+        Returns if the Weierstrass isomorphims is the identity.
```

Comment:
Docstrings start with "Return" (no s)

Reply:
Also typo. I recommend rewriting as "Check if this Weierstrass isomorphism is the identity." ("check" and "return" are common verbs in Sage docs)